### PR TITLE
Pretrained resnet50

### PR DESCRIPTION
**Pull request by ekatiyar**
@Shaviv-Hoffman-Lowitz  can u comment ur accuracy and num epochs here

### CODE REVIEW

**Review by Shaviv-Hoffman-Lowitz — APPROVED** [review]
Accuracy: 40.6%, num epochs: 2

**Comment by wwang3990** [comment]
Looks good to me, nice work!

**Review by wwang3990 — APPROVED** [review]
Looks good, we might want to upload the trained models to github for future submissions.
